.github/workflows/redis-docker.yml

```
@@ -60,15 +68,15 @@ jobs:
           platforms: linux/amd64,linux/arm64
 
       - name: Aqua Security Trivy
-        uses: aquasecurity/trivy-action@0.28.0
+        uses: aquasecurity/trivy-action@915b19bbe73b92a6cf82a1bc12b087c9a19a5fe2 # 0.28.0
         with:
             image-ref: nataliagranato/redis:${{ steps.generate-tag.outputs.tag }}
```

yamllint

[warning] wrong indentation: expected 10 but found 12

(indentation)